### PR TITLE
Add account/upload endpoint

### PR DESCRIPTION
**Pull request by dosvath**
## Problem

No endpoint to upload user documents.

## Solution

Add endpoint to upload user documents.

## Testing Done

- [x] Spec builds using `redoc-cli serve`
- [x] Previewed changes locally

### CODE REVIEW

File: openapi/openapi.yml [hunk 1]
```diff
@@ -392,6 +392,61 @@ paths:
...
+        - Account Opening
+      operationId: uploadDocuments
+      summary: Upload user documents for account creation.
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                mentum_user_id:
```
Comment (ncassata) [inline openapi/openapi.yml:407]: Should this data be required?

**Review by estevaofay — APPROVED** [review]
LGTM!

File: openapi/openapi.yml (hunk 1 above)
Comment (dosvath) [reply openapi/openapi.yml:407]: Great point I thought `multipart/form-data` would make all required by default, but it seems not, I'll update.

**Review by ncassata — APPROVED** [review]
LGTM. Thanks for those revisions.

**Pull request merged**